notebook/automl_xgboost.ipynb

```diff
@@ -29,7 +29,7 @@
     "\n",
     "FLAML requires `Python>=3.7`. To run this notebook example, please install flaml with the `notebook` option:\n",
     "```bash\n",
-    "pip install flaml[notebook]\n",
+    "pip install flaml[notebook]==1.0.8\n",
```

Comment:
is there a reason why this version is 1.0.8 and not 1.1.2 since all the other ones are 1.1.2

Reply:
Good point, not sure why. I'll bump it. Thanks!

Reply:
I've bumped it now and pushed